publish.gradle

```diff
@@ -30,7 +30,6 @@ afterEvaluate {
                 version = "$versions.retromock"
 
                 artifact sourceJar
-                artifact javadocJar
 
                 pom {
                     name.set("Retromock")
```

Comment:
Can't we just apply project level publish file?

Reply:
Just for feature reasons, in case we have different modules that can be published, so I just went with our current setup